docs/knowledgebase/getting-started/glossary.md

```diff
@@ -106,75 +133,89 @@ is imperfect information about whether a component has failed) in systems that r
 
 ## Consensus
 
-The process by which validators agree upon the next valid block to be added to the relay chain.
+The process by which [authorities](#authority) agree upon the next valid [block](#block) to be added
+to a [blockchain](#blockchain).
 
 ### Consensus Engine
 
-A means for forming consensus over what constitutes the (one true) canonical chain. In the context
-of Substrate, it refers to a set of trait implementations that dictate which of a number of input
-blocks form the chain.
+A means for forming consensus over what constitutes the canonical (true) chain. In the context of
+Substrate, it refers to a set of algorithms that dictate which of a number of input [blocks](#block)
+form the [blockchain](#blockchain).
 
 ### Consensus Algorithms
 
-An algorithm that is able to ensure that a set of actors who don't necessarily trust each other can
-come to consensus over some computation. Often mentioned alongside "safety" (the ability to ensure
-that any progression will eventually be agreed as having happened by all honest nodes) and
-"liveness" (the ability to keep making progress).
+An algorithm that is able to ensure that a set of [actors](#authority), who don't necessarily trust
+each other, can come to consensus over some computation. Often mentioned alongside "**safety**" (the
+ability to ensure that any progression will eventually be agreed as having happened by all honest
+nodes) and "**liveness**" (the ability to keep making progress). Many consensus algorithm are
+referred to as being [Byzantine-fault-tolerant](#byzantine-fault-tolerance-bft).
 
-See
-https://medium.com/polkadot-network/grandpa-block-finality-in-polkadot-an-introduction-part-1-d08a24a021b5
+There is
```

Comment:
@danforbes Use my feedback on the previous two entries (consensus, and consensus engine) as you see fit.

Reply:
Honestly I'm inclined to strip these definitions way down. If we define "consensus" well, it shouldn't be super necessary to elaborate extensively on what a "consensus engine" or "consensus algorithm" is.